dashboard/tasks_test.py

```diff
@@ -60,13 +59,7 @@ def test_batch_update_user_date_for_active_users(self, mocked_subtasks):
         for user in self.all_working_users:
             user.is_active = False
             user.save()
-        now = now_in_utc()
-        UserCacheRefreshTimeFactory.create(
-            user=self.user2,
-            enrollment=now,
-            certificate=now,
-            current_grade=now,
-        )
+        UserCacheRefreshTimeFactory.create(user=self.user2, unexpired=True)
```

Comment:
I think this should be `expired=True` since `now` would have become the past in the time that `batch_update_user_data` is called. I'm not sure it matters for this test though